LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/Common/CarePackageSchedulingOption.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.Common
+{
+    public class CarePackageSchedulingOption
+    {
+        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
```

Comment:
Since we're already using enum in domain why not use it in entity?

Reply:
This was added in just as a lookup for anyone working from DB. The enum is used in care package if you check again

Reply:
Yes, I see, and question is the same - why not use same type for same things in .NET code :)